scripts/akrtma3d/exakrtma3d_obsprep_radar.ksh

```diff
@@ -123,6 +123,7 @@ if [ -s filelist_mrms ]; then
    mv filelist_mrms filelist_mrms_org
    ls MergedReflectivityQC_*_${YYYY}${MM}${DD}-${HH}????.grib2 > filelist_mrms
    numgrib2=`more filelist_mrms | wc -l`
+   cpreq MergedReflectivityQC_*_${YYYY}${MM}${DD}-${HH}????.grib2 ${COMINobsproc_rtma3d}
```

Comment:
Shouldn't we copy the gzip'd files? The files would be smaller and that is what the script is looking for if we do a rerun.